airbyte-integrations/connectors/source-discourse/integration_tests/catalog.json

```diff
@@ -0,0 +1,39 @@
+{
+  "streams": [
```

Comment:
you can remove this file. Btw @natalyjazzviolin can you create an issue to remove his from generator?

Reply:
@marcosmarxm done!